lib/crowbar/init/application.rb

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+require "/opt/dell/crowbar_framework/lib/crowbar/upgrade_status"
```

Comment:
The installation path `/opt/dell/...` is a decision which should be made by the packager, not the developer.  The code should be relocatable, so that we're not creating extra work for ourselves for SOC8 when we finally ditch use of the embarrassing `/opt/dell` prefix.